toolkit/docs/building/building.md

```diff
@@ -95,87 +103,48 @@ cd CBL-Mariner/toolkit
-
-```bash
-# Build an image without downloading pre-compiled packages
-sudo make image REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64" -j$(nproc)
-```
+**IMPORTANT:** The 1.0-stable tag always points to the latest known good build of CBL-Mariner. At this time, only the Mariner 1.0 branch is buildable.  This branch is continually updated with bug fixes, security vulnerability fixes or occassional feature enhancements.  Fixes may be applied to this branch at any time.  As those fixes are integrated into the branch the head of a branch may be temporarily unstable.  The 1.0-stable tag will remain fixed until the tip of the branch is validated and the latest source and binary packages (SRPMs and RPMs) are published.  At that point, the 1.0-stable tag is advanced.  
 
-## Further Reading
+It is also possible to build an older version of CBL-Mariner from the 1.0 branch.  CBL-Mariner may be updated at any time, but an aggregate release is declared monthly and [tagged in github](https://github.com/microsoft/CBL-Mariner/releases).  These monthly builds are stable and their tags can be substituted for the 1.0-stable label above.
 
-### Building in Stages
+Alternate branches are not generally buildable because community builds require the SRPMs and/or RPMs be published.  At this time, published files are only available for the 1.0 branch.
 
-This section runs through a build one step at a time, briefly explaining the purpose. `Make` will generally automate this flow if given a target, however it can be useful for debugging.
+**NOTE: All subsequent commands are assumed to be executed from inside the toolkit directory.**
 
-#### 1) Bootstrap Toolchain
+## **Toolchain Stage**
 
-A set of bootstrapped toolchain packages (gcc etc.) are used to build CBL-Mariner packages and images.
+The toolchain builds in two sub-phases.  The first phase builds an initial _bootstrap_ toolchain which is then used to build the _final_ toolchain used in package building.  In the first phase, the bootstrap toolchain downloads a series of source packages from upstream sources.  The second phase downloads SRPMS from packages.microsoft.com.
 
-```bash
-# Download the pre-built toolchain packages (REBUILD_TOOLCHAIN=n is the default value)
-sudo make toolchain REBUILD_TOOLS=y
+For expediency, the toolchain may be populated from upstream binaries, or may be completely rebuilt. 
 
-# Move the downloaded bootstrap packages to the general RPM out folder
-sudo make copy-toolchain-rpms
-```
+### **Populate Toolchain**
 
-NOTE: A full list of targets and options to `make` is available [here](#all-build-targets) and [here](#all-build-variables).
+A set of bootstrapped toolchain packages (gcc etc.) are used to build CBL-Mariner packages and images.  Rather than built the toolchain, the prebuilt binaries can be downloaded to your local machine.
```

Comment:
>the prebuilt binaries can be downloaded to your local machine. [](start = 137, length = 62)

Might be a good place to explain how to download the prebuilt toolchain and how to apply it.

Reply:
Should be as simple as not setting `REBUILD_TOOLCHAIN`, download is the default behaviour here. Still might be good to call this out.

Reply:
agreed.  added short explanation and example to do it.